react/react1/week3/todo-app-week3/src/components/Form.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+const Form = ({
+  setInput,
+  todos,
+  setTodos,
+  input,
+  deadlineValue,
+  setDeadlineValue,
+}) => {
+  const inputHandler = (e) => {
+    setInput(e.target.value);
+  };
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    const newTodo = {
+      id: todos.length + 1,
```

Comment:
Here is the same bug as in the previous task
And and if you delete one of the ID 3, then both will be deleted

![image](https://user-images.githubusercontent.com/73549460/197746876-61b77eba-2292-40c9-9b7f-b42e2de16ace.png)